jdk/make/CopyFiles.gmk

```diff
@@ -1,12 +1,13 @@
-#
+# ===========================================================================
 # Copyright (c) 2011, 2016, Oracle and/or its affiliates. All rights reserved.
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
```

Comment:
Please add the IBM copyright above the Oracle copyright, and do not change the license. See jdk/make/CompileLaunchers.gmk for an example.

Reply:
Already updated.